build.gradle

```diff
@@ -125,6 +125,11 @@ def resolveLargeResourceStubFiles(largeResourcesFolder, buildPrerequisitesMessag
     }
 }
 
+
+def looksLikeWereInAGitRepository(){
```

Comment:
Might be worth putting a comment here saying that its looking for either a git-dir or a worktree (or whatever the right names are) - I always get them mixed up.